### PR TITLE
CertificateS error message (EXPOSUREAPP-8226)

### PR DESCRIPTION
**Pull request by mtwalli**
Screenshot 2021-07-02 at 15 26 00

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
     <string name="vaccination_card_status_vaccination_complete">"Gültig bis einschließlich %s"</string>
 
     <!-- XTXT: Vaccination QR code scan error message-->
-    <string name="error_vc_invalid">Dieser QR-Code ist kein gültiges Impfzertifikat.\n\nWeitere Informationen zum Erhalt Ihres Impfzertifikats finden Sie in den FAQ.</string>
+    <string name="error_vc_invalid">Dieser QR-Code ist kein gültiges Zertifikat.\n\nWeitere Informationen zum Erhalt Ihres Zertifikats finden Sie in den FAQ.</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml:99]: @NataliaLemmerth  Please let me know what is the right translation?

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml (hunk 1 above)
Comment (NataliaLemmerth) [reply Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml:99]: @mtwalli It's correct. Thank you very much!

**Comment by dsarkar** [comment]
@NataliaLemmerth @mtwalli 
@MikeMcC399 pointed pointed out that this string still need to be taking care of:

https://github.com/corona-warn-app/cwa-app-android/blob/d08d828f219ed0bedb5a4903ee78b60c3a7bda4d/Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml#L109-L110

    
    <string name="error_button_vc_faq">FAQ zu Impfzertifikaten</string>

**Comment by dsarkar** [comment]
@NataliaLemmerth @mtwalli @nickguendling Also, see https://github.com/corona-warn-app/cwa-app-ios/pull/3070/files
iOS error code is **HC_**PREFIX_INVALID, not PREFIX_INVALID

**Review by harambasicluka — APPROVED** [review]

**Comment by mtwalli** [comment]
> @NataliaLemmerth @mtwalli
> @MikeMcC399 pointed pointed out that this string still needs to be taking care of:
> 
> https://github.com/corona-warn-app/cwa-app-android/blob/d08d828f219ed0bedb5a4903ee78b60c3a7bda4d/Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml#L109-L110
> 
> ```
> 
> <string name="error_button_vc_faq">FAQ zu Impfzertifikaten</string>
> ```

@dsarkar  it is done here https://github.com/corona-warn-app/cwa-app-android/pull/3617/files#diff-fcc1b466984b0efa9d6f2acb9a2d2f9a16b4251afd27e101a23b7972fd6c3728R110

**Pull request merged**